main.go

```diff
@@ -5,6 +5,12 @@ package main
 import (
 	"flag"
 	"fmt"
+	"os"
+	"os/signal"
+	"regexp"
+	"sync"
+	"syscall"
+
```

Comment:
Does this really comply to go fmt?

Reply:
As long as there is a empty line between the imports from the stdlib and the other imports, yes. I removed the empty line and `go fmt` sorted it alphabetical. But this does not comply to `goimports` for example.